server.js

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
```

Comment:
[It's the default value for `npm start`](https://docs.npmjs.com/cli/start). Maybe it's not too much known and people use other alternatives, but it's the standard one.